src/main.tsx

```diff
@@ -30,6 +31,7 @@ const router = createBrowserRouter([
 
 // Register demo feature flags
 registerDemoFeatureFlags();
+registerFeatureFlags();
```

Comment:
this needs to be inside `src/core/main/index`
Main.tsx is not bundled when we deploy the pacakge